packages/fx-core/src/plugins/resource/identity/constants.ts

```diff
@@ -46,7 +46,7 @@ export class IdentityBicep {
   static readonly identityClientId: string =
     "provisionOutputs.identityOutput.value.identityClientId";
   static readonly identityResourceId: string =
-    "userAssignedIdentityProvision.outputs.identityResourceId";
```

Comment:
@xzf0587  this constants value needs to be updated, right ?

Reply:
yes. please update it.